app/src/main/java/org/fossasia/pslab/fragment/ControlFragmentRead.java

```diff
@@ -81,24 +81,24 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         buttonControlRead1.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                /*double resistance = scienceLab.getResistance();
-                tvControlRead1.setText(String.valueOf(resistance));*/
+                double resistance = scienceLab.getResistance();
```

Comment:
same ```sciencelab.isConnected()``` check here and all other such places where sciencelab methods are accessed.